Makefile

```diff
@@ -179,7 +179,7 @@ test_cover:
 
 test_lint:
 	gometalinter.v2 --config=tools/gometalinter.json ./...
-	!(gometalinter.v2 --disable-all --enable='errcheck' --vendor ./... | grep -v "client/")
+	!(gometalinter.v2 --disable-all --enable='errcheck' --vendor ./... | grep -v -e "client/" -e "fmt\.Fprintf")
```

Comment:
This has since been fixed, and this is the wrong way to fix it anyways (the linter is right) - https://github.com/cosmos/cosmos-sdk/pull/2257

Reply:
Well, by definition linter is correct, but I wonder how helpful it would be to check errors reported by `fprintf()` :). I'll revert my changes nonetheless, thanks